### DISCUSSION TITLE
Simple way to draw leading spaces on my code

### DISCUSSION THREAD

**Discussion by lucastrvsn** [neovim/neovim#13859] [Q&A] [answered]
Hello guys, how you doing?

I'm trying to show leading spaces on my code without need to use a whole plugin to do it. I can draw leading spaces using some built-in function that I don't know in neovim? I was trying to use `Conceal` but no success. 

This is my attempt:

```vim
let leading_space_char="·"
call matchadd('Conceal', '\%(^\s*\)\@<= ', 0, -1, { 'conceal': leading_space_char })
```

Another question: can I use `blend` option on the leading char space? Can someone help me with this? Thanks a lot!

(I'm using the latest version of neovim master branch)

**Comment by chrisbra** [comment]
you might need to enable concealing, e.g. `:set conceallevel=2 concealcursor=nv`

**Comment by xbc5** [comment, marked as answer]
`set list` will show all invisible chars, including CR and spaces.

**Reply by justinmk** [reply]
Also look at `:help lcs-lead`.
